spec/board_spec.rb

```diff
@@ -1,12 +1,18 @@
 require 'board'
 
 describe Board do
+
+  before(:each) do
+    @consoleInOut = ConsoleInOut.new('', '')
```

Comment:
In your tests `ConsoleInOut` takes Strings. But in the real version, it takes STDIN/STOUT. 

Those things are not the same. Have a look at StringIO.